importer/test_main.py

```diff
@@ -18,6 +21,36 @@ def test_read_csv(self):
         self.assertIsInstance(records, list)
         self.assertEqual(len(records), 3)
 
+    def test_write_csv(self):
+        self.test_data = [
+            [
+                "e2e-mom",
+                "True",
+                "update",
+                "caffe509-ae56-4d42-945e-7b4c161723d1",
+                "d93ae7c3-73c0-43d1-9046-425a3466ecec",
+                "handy",
+            ],
+            [
+                "e2e-skate",
+                "True",
+                "update",
+                "2d4feac9-9ab5-4585-9b33-e5abd14ceb0f",
+                "58605ed8-7217-4bf3-8122-229b6f47fa64",
+                "foolish",
+            ],
+        ]
+        self.test_resource_type = "test_organization"
+        self.test_fieldnames = ["name", "active", "method", "id", "identifier", "alias"]
+        write_csv(self.test_data, self.test_resource_type, self.test_fieldnames)
+        self.assertIsInstance(self.test_data, list)
+        self.assertEqual(len(self.test_data), 2)
+        current_time = datetime.now().strftime("%Y-%m-%d-%H-%M")
+        expected_csv_file_path = (
+            f"csv/exports/{current_time}-export_{self.test_resource_type}.csv"
```

Comment:
It is possible that this current_time gotten in the test is different from the current_time gotten when the function was called